ambitious/block-template-parts/front-page/front-page-intro.html

```diff
@@ -0,0 +1,33 @@
+<!-- wp:group {"backgroundColor":"primary","textColor":"white","className":"front-page-intro"} -->
+<div class="wp-block-group has-primary-background-color has-text-color has-background front-page-intro">
+	<div class="wp-block-group__inner-container">
+
+		<!-- wp:heading {"align":"center","level":1,"textColor":"white","className":"intro-heading has-huge-font-size"} -->
+		<h1 class="has-white-color has-text-color has-text-align-center intro-heading has-huge-font-size">
+			Build your site with Blocks
+		</h1>
+		<!-- /wp:heading -->
+
+		<!-- wp:paragraph {"align":"center","textColor":"white","fontSize":"medium"} -->
+		<p class="has-text-color has-text-align-center has-medium-font-size has-white-color">
+			Ambitious is an experimental block-based WordPress Theme. Open your Browser DevTools to see the header, navigation, content and footer made out of Gutenberg blocks entirely.
+		</p>
+		<!-- /wp:paragraph -->
+
+		<!-- wp:buttons {"align":"center"} -->
+		<div class="wp-block-buttons aligncenter">
+
+			<!-- wp:button {"textColor":"white","borderRadius":0,"className":"is-style-outline"} -->
+			<div class="wp-block-button is-style-outline"><a class="wp-block-button__link has-text-color has-white-color no-border-radius" href="https://github.com/Netzberufler/ambitious/releases">Download Theme</a></div>
+			<!-- /wp:button -->
+
+			<!-- wp:button {"textColor":"white","borderRadius":0,"className":"is-style-outline"} -->
+			<div class="wp-block-button is-style-outline"><a class="wp-block-button__link has-text-color has-white-color no-border-radius" href="https://github.com/Netzberufler/ambitious">Github Repository</a></div>
+			<!-- /wp:button -->
+
+		</div>
+		<!-- /wp:buttons -->
+
+	</div>
+</div>
+<!-- /wp:group -->
```

Comment:
I ran into this same use case of wanting to include content in block templates. 

Since it's not re-used anywhere, I think this belongs in starter content rather than as a block template part. 

Some related context in the Gutenberg repo: https://github.com/WordPress/gutenberg/issues/18055

cc @kjellr

Reply:
Ah okay, good to know. I saw the about.html and contact.html files in the Parisienne theme and thought theme authors can now directly provide block templates for starter content.

I think this actually would be a really great feature. So much easier to provide both starter content and a predefined block layout in just one template file, instead of having two separate places. I will check out the issue on the Gutenberg project about that, haven't seen that one yet.

Reply:
> I saw the about.html and contact.html files in the Parisienne theme and thought theme authors can now directly provide block templates for starter content.

Yeah, I think this would be a great improvement, either through "Starter Page Templates" as described above, or through something like what was explored in https://github.com/WordPress/theme-experiments/pull/16.